docs/xls-33d-multi-purpose-tokens/reference/mptoken.md

```diff
@@ -44,8 +44,8 @@ The `MPTokenID` is the result of SHA512-Half of the following values, concatenat
 | LedgerEntryType   | number    | UInt16        | The value 0x007F, mapped to the string `MPToken`, indicates that this object describes an individual account's holding of an MPT. |
 | `Account`           | string    | AccountID     | The owner of the MPT. |
 | `MPTokenIssuanceID` | string    | UInt256       | The `MPTokenIssuance` identifier. |
-| `MPTAmount`         | string    | UInt64        | This value specifies a positive amount of tokens currently held by the owner. Valid values for this field are between 0x0 and 0xFFFFFFFFFFFFFFFF. |
-| `LockedAmount`      | string    | UInt64        | (Default) Specifies a positive amount of tokens that are currently held in a token holder's account but that are unavailable to be used by the token holder. Locked tokens might, for example, represent value currently being held in escrow, or value that is otherwise inaccessible to the token holder. The initial value is 0 in order to save space on the ledger for an empty MPT holding. |
+| `MPTAmount`         | string    | Base10        | This value specifies a positive amount of tokens currently held by the owner. Valid values for this field are between 0x0 and 18,446,744,073,709,551,615. |
```

Comment:
@shawnxie999 any opinions on the internal name type here?  should it be `Amount` or `Base10`?

Reply:
If it's not one of the types in the [type list](https://xrpl.org/docs/references/protocol/binary-format#type-list) then either it's new and will need to be added to that list, or it's named wrong.

I think the internal type is still UInt64 but maybe I'm wrong on that and it's actually the [new STNumber type](https://github.com/XRPLF/rippled/pull/5121)?

Reply:
Internal type is still UInt64. The base-10 serialization is special-cased for a few [fields](https://github.com/XRPLF/rippled/pull/5143/files#diff-2ed7b91d9113cc839eab5e56e31cc3c3501a89763745d5491e3c46791d44247bR208-R213).

Reply:
it should remain UInt64